src/routes/files/handlers.test.js

```diff
@@ -7,6 +7,9 @@ import db, {
 } from '../../models';
 import app from '../../app';
 import s3Uploader from '../../lib/s3Uploader';
+// disable this to allow mocking this function
+// eslint-disable-next-line no-unused-vars
```

Comment:
Instead of disabling this check, could we add a test to verify that `addToScanQueue` is called? I think that would be a valuable test and also avoid the linter error.